### ISSUE TITLE
'panic: interface conversion' when trying to convert oteemocharts/sonarqube

### ISSUE THREAD

**Issue by eyalzek** [ContainerSolutions/helm-convert#30]
Attempted using the docker image or on my host machine running debian `bullseye/sid`. To reproduce:
```bash
$ helm repo add oteemocharts https://oteemo.github.io/charts
$ helm convert --destination /tmp/sonarqube --name sonarqube  oteemocharts/sonarqube
panic: interface conversion: interface {} is nil, not []interface {}

goroutine 1 [running]:
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).getImageTag(0xc0002bc740, 0xc0003cc160, 0xc000597ec0, 0xd0ef84, 0xe, 0x1, 0x0)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:60 +0x513
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).findImage(0xc0002bc740, 0xc0003cc160, 0xc000597ec0, 0x4, 0xc000214998)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:47 +0x16d
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).findContainers(0xc0002bc740, 0xc0003cc160, 0xc000597e30, 0xe, 0x143c060)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:109 +0x20c
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).findImage(0xc0002bc740, 0xc0003cc160, 0xc000597e30, 0x8, 0xc0002142f8)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:54 +0x100
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).findContainers(0xc0002bc740, 0xc0003cc160, 0xc000597da0, 0xe, 0x143c060)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:109 +0x20c
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).findImage(0xc0002bc740, 0xc0003cc160, 0xc000597da0, 0x4, 0xc0001b7f18)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:54 +0x100
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).findContainers(0xc0002bc740, 0xc0003cc160, 0xc000597d10, 0xe, 0x143c060)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:109 +0x20c
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).findImage(0xc0002bc740, 0xc0003cc160, 0xc000597d10, 0xc000267f68, 0x0)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:54 +0x100
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*imageTransformer).Transform(0xc0002bc740, 0xc0003cc160, 0xc0001e4010, 0x0, 0x0)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/image.go:28 +0x12c
github.com/ContainerSolutions/helm-convert/pkg/transformers.(*multiTransformer).Transform(0xc0002bc760, 0xc0003cc160, 0xc0001e4010, 0x8, 0xc0005e5b28)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/pkg/transformers/multitransformer.go:27 +0x79
github.com/ContainerSolutions/helm-convert/cmd.(*convertCmd).run(0xc00018c180, 0xc0000fbd88, 0xc0000fbd58)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/cmd/convert.go:255 +0xc3b
github.com/ContainerSolutions/helm-convert/cmd.NewConvertCommand.func2(0xc00014a280, 0xc0000b0c80, 0x1, 0x5, 0x0, 0x0)
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/cmd/convert.go:106 +0x99
github.com/spf13/cobra.(*Command).execute(0xc00014a280, 0xc0000321f0, 0x5, 0x5, 0xc00014a280, 0xc0000321f0)
	/home/travis/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0xc00014a280, 0xd63ee0, 0xc0000fbf88, 0xc0000380b8)
	/home/travis/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/home/travis/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main()
	/home/travis/gopath/src/github.com/ContainerSolutions/helm-convert/main.go:13 +0x44
Error: plugin "convert" exited with error
```

**Comment by etiennetremel** [comment]
@eyalzek fixed in 0.5.2: https://github.com/ContainerSolutions/helm-convert/releases/tag/v0.5.2

**Comment by eyalzek** [comment]
Thanks for the quick fix!
